frontend/packages/console-dynamic-plugin-sdk/src/api/host-app-api.ts

```diff
@@ -0,0 +1,11 @@
+export { default as AppInitSDK } from '@console/dynamic-plugin-sdk/src/app/AppInitSDK';
+export { default as coreReducer } from '@console/dynamic-plugin-sdk/src/app/core/reducers/core';
+export {
+  setUser,
+  beginImpersonate,
+  endImpersonate,
+} from '@console/dynamic-plugin-sdk/src/app/core/actions/core';
+export {
+  getUser,
+  getImpersonate,
+} from '@console/dynamic-plugin-sdk/src/app/core/reducers/coreSelector';
```

Comment:
Isn't this backwards? Shouldn't our host-app-api be in here and exported via the other files?

Reply:
we are exporting as a new package here so, this should be its own export for host-app as we have for `core-api` , `internal-kubevirt` etc as in her https://github.com/openshift/console/tree/master/frontend/packages/console-dynamic-plugin-sdk/src/api

let me know if am missing something here.

Reply:
You are quite right... apologizes. I thought of us making this where things were defined, but it makes more sense the way you have it. They are exposed where they need to be exposed, but written in a single spot.